docs/guide/apm-data-model.asciidoc

```diff
@@ -144,4 +157,100 @@ For a full list of tracked metrics, see the relevant agent documentation:
 * {apm-java-ref-v}/metrics.html[Java]
 * {apm-node-ref-v}/metrics.html[Node.js]
 * {apm-py-ref-v}/metrics.html[Python]
-* {apm-ruby-ref-v}/metrics.html[Ruby]
\ No newline at end of file
+* {apm-ruby-ref-v}/metrics.html[Ruby]
+
+// This heading is linked to from the APM UI section in Kibana
+[[metadata]]
+=== Metadata
+
+Metadata can enrich your events and make application performance monitoring even more useful.
+Let's explore the different types of metadata that Elastic APM offers.
+
+==== Quick Reference table
```

Comment:
Probably just going to move this information into the sections below. I fear people will skip over the main sections with this here.